chorus_book/src/guide-projector.md

```diff
@@ -4,19 +4,21 @@ Projector is responsible for performing the end-point projection and executing t
 
 ## Creating a Projector
 
-To create a `Projector`, you need to provide the location and the transport.
+To create a `Projector`, you need to provide the set of locations it can work with, the target location, and the transport.
```

Comment:
Maybe mention that we use `project!` macros instead of directly instantiating `Projector`

Reply:
done.